types/plan/planutils.go

```diff
@@ -30,14 +30,14 @@ import (
 func ReadPlan(path string, sourceDir string) (Plan, error) {
 	plan := Plan{}
 	var err error
+	absSourceDir := ""
 	if err = common.ReadMove2KubeYaml(path, &plan); err != nil {
 		logrus.Errorf("Failed to load the plan file at path %q Error %q", path, err)
 		return plan, err
 	}
-	if sourceDir != "" {
-		plan.Spec.SourceDir = sourceDir
+	if plan.Spec.SourceDir != "" {
+		absSourceDir, err = filepath.Abs(plan.Spec.SourceDir)
```

Comment:
Why are we ignoring the use of sourceDir here?

Reply:
Updated it1